### DISCUSSION TITLE
SMTP+Docker+GMail

### DISCUSSION THREAD

**Discussion by mmol67** [Leantime/leantime#1362] [General] [answered]
Hello. 

I'm beginning with leantime and  I'm trying to get this working using Portainer stacks.

These are my env variables for email

```
      LEAN_EMAIL_RETURN: 'myuser@gmail.com'
      LEAN_EMAIL_USE_SMTP: 'true'
      LEAN_EMAIL_SMTP_HOSTS: 'smtp.gmail.com'
      LEAN_EMAIL_SMTP_USERNAME: 'myuser@gmail.com'
      LEAN_EMAIL_SMTP_PASSWORD: 'mypass'
      LEAN_EMAIL_SMTP_AUTO_TLS: 'true'
      LEAN_EMAIL_SMTP_SECURE: 'STARTTLS'   # tried 'TLS' too
      LEAN_EMAIL_SMTP_PORT: 587
```
Then I created a user, and invited him.

I've read the topics in there for this, but nothing seems to work.

I've received no email. What am I missing/ doing wrong?

And, there is a way to set the user password without using the e-mail?

**Comment by TareqK** [comment, marked as answer]
For Gmail specifically, you need to enable less secure apps for the user to
allow login via password on plain SMTP

**Comment by mmol67** [comment]
Thank you! Fixed. I needed to activate 2 pass authentication and create an application pass for leantime and it worked.
